src/Spectre.libGenetic.Tests/CrossoverOperatorTest.cpp

```diff
@@ -41,25 +49,25 @@ class CrossoverOperatorTest: public ::testing::Test
     const Seed SEED = 0;
     const Individual true_individual = Individual({ true, true, true, true });
     const Individual false_individual = Individual({ false, false, false, false });
-    CrossoverOperator crossover;
+    std::unique_ptr<CrossoverOperator> crossover;
 
     void SetUp() override
     {
-        crossover = CrossoverOperator(SEED);
+        crossover = std::make_unique<CrossoverOperator>(SEED);
     }
 };
 
 TEST_F(CrossoverOperatorTest, throws_on_inconsistent_sizes)
 {
     const Individual shorter({ false });
     const Individual longer({ false, true });
-    EXPECT_THROW(crossover(shorter, longer), InconsistentChromosomeLengthException);
+    EXPECT_THROW(crossover->operator()(shorter, longer), InconsistentChromosomeLengthException);
```

Comment:
Do whatever you feel is appropriate, but personally I believe that ```crossover->operator()(shorter, longer)``` looks a bit hacky.
Why not to try ```(*crossover.get())(shorter, longer)```? It's a bit shorter and to me it makes much more sense to extract the pointer, dereference it and perform the call.

Reply:
Maybe you're right, personally don't mind it ;], so I changed it. Done